recipes/sepp/meta.yaml

```diff
@@ -1,11 +1,11 @@
-{% set version = "4.5.1" %}
-{% set sha256 = "51e052569ae89f586a1a94c804f09fe1b7910a3ffff7664e2005f18c7d3f717b" %}
+{% set version = "4.5.4" %}
```

Comment:
This comment https://github.com/bioconda/bioconda-recipes/pull/48294#issuecomment-2233974593 mentions needing a new major release. But it seems @smirarab decided to release 4.5.4 with information for the user on hmmer version differences as mentioned this instead in comment https://github.com/smirarab/sepp/issues/76#issuecomment-2284322297

Can you just confirm I'm understanding the situation correctly?

Reply:
@smirarab and I had some back and forth discussion via email to come up with a suitable solution for the stand alone SEPP version to stick to the bundled, fixed binaries of HMMER and flexible HMMER versions for the bioconda package. The recipe - when I created it back in 2018 - always ignored the bundled binaries, but was fixed to the same version. As QIIME2 is going to upgrade to py3.10 we need more recent HMMER versions but this will slightly change output of SEPP. 
Therefore, the user will hopefully notice these differences (stand-alone VS conda) by looking at the new HMMER version information line. Thus, the change in upstream SEPP does not really qualify for a major version change.

Reply:
Make sense, thanks.